src/main/java/com/amadeus/airport/DirectDestinations.java

```diff
@@ -0,0 +1,60 @@
+package com.amadeus.airport;
+
+import com.amadeus.Amadeus;
+import com.amadeus.Params;
+import com.amadeus.Response;
+import com.amadeus.exceptions.ResponseException;
+import com.amadeus.resources.DirectDestination;
+import com.amadeus.resources.Resource;
+
+/**
+ * <p>
+ *   A namespaced client for the
+ *   <code>/v1/airport/direct-destinations</code> endpoints.
+ * </p>
+ *
+ * <p>
+ *   Access via the Amadeus client object.
+ * </p>
+ *
+ * <pre>
+ * Amadeus amadeus = Amadeus.builder("clientId", "secret").build();
+ * amadeus.airport.directDestinations;</pre>
+ */
+public class DirectDestinations {
+  private final Amadeus client;
+
+  /**
+   * Constructor.
+   * @hide
+   */
+  public DirectDestinations(Amadeus client) {
+    this.client = client;
+  }
+
+  /**
+   * <p>
+   *    Find all destinations served by a given airport.
+   * </p>
+   *
+   * <pre>
+   * amadeus.airport.directDestinations.get(Params.with("origin", "LHR"));
```

Comment:
the required parameter is `departureAirportCode` according to the [docs](https://developers.amadeus.com/self-service/category/air/api-doc/airport-routes/api-reference)